packages/front-end/components/Experiment/BreakDownResults.tsx

```diff
@@ -140,27 +152,37 @@ const BreakDownResults: FC<{
         const ret = sortAndFilterMetricsByTags([metric], metricFilter);
         if (ret.length === 0) return;
 
-        const { newMetric } = applyMetricOverrides(metric, metricOverrides);
+        const { newMetric, overrideFields } = applyMetricOverrides(
+          metric,
+          metricOverrides
+        );
         let metricSnapshotSettings: MetricSnapshotSettings | undefined;
         if (settingsForSnapshotMetrics) {
           metricSnapshotSettings = settingsForSnapshotMetrics.find(
             (s) => s.metric === metricId
           );
         }
 
+        const rows: ExperimentTableRow[] = results.map((d) => ({
+          label: d.name,
+          metric: newMetric,
+          variations: d.variations.map((variation) => {
+            return variation.metrics[metricId];
+          }),
+          metricSnapshotSettings,
+          resultGroup: getMetricResultGroup(
+            metricId,
+            goalMetrics,
+            secondaryMetrics
+          ),
```

Comment:
Nit: This can be done outside of the map.  Also, if we do that, we can use the result of this for `isGuardrail` below instead of duplicating the logic.